static/js/invite.js

```diff
@@ -34,7 +34,12 @@ function reset_error_messages() {
 
 function get_common_invitation_data() {
     const invite_as = Number.parseInt($("#invite_as").val(), 10);
-    const expires_in = Number.parseFloat($("#expires_in").val());
+    let expires_in = $("#expires_in").val();
+    if (expires_in !== "null") {
```

Comment:
I think this check should be for the value `null`, not the string?

Reply:
Yes, this is now `null` as we have updated the value sent from API to `None` as you suggested in one of the above comments.